sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json

```diff
@@ -26,13 +26,14 @@
     "sunpy.map.tests.test_plotting.test_rectangle_aia171_width_height": "455246d338564add5be94c1dc0ff5740a90d21d8ef7da9f392ba3a01237ef5b8",
     "sunpy.timeseries.tests.test_timeseriesbase.test_esp_peek": "b9a0d6b7c14fe6175e4875385f3ccfb2ddffdea092792997db30066357b7ac68",
     "sunpy.timeseries.tests.test_timeseriesbase.test_eve_peek": "fe84a95c9943554660acdf5202cf68ffc13333c5db29cf5ebb1d4f296043c882",
-    "sunpy.timeseries.tests.test_timeseriesbase.test_fermi_gbm_peek": "a0addeae9ec0eda509c976e9058b2d90fa748b2c3d4a64df43b3a064d85f34d9",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_fermi_gbm_peek": "5cbe52c6cd266ff85e599390979a5c431304212bd38dd112b0eaf4be31488e44",
```

Comment:
Don't merge with this changed!! See https://github.com/sunpy/sunpy/issues/4342

Reply:
Should I undo these changes and leave these tests failing for now? I would rather not wait on fixing the dev figure tests.

Reply:
yeah from the issue looks like this should be reverted if it hasn't been already?

Reply:
I have reverted it.